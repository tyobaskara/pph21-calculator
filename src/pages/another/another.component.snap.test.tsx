import React from "react";
import renderer from 'react-test-renderer';
import AnotherPage from './another.component';

jest
  .mock('../../components', () => ({
    Navigation: 'NavigationComponent'
  }));

test('AnotherPage snaptest', () => {
  const component = renderer.create(
    <AnotherPage/>
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});