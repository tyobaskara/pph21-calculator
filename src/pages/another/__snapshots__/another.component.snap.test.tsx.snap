// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AnotherPage snaptest 1`] = `
<div>
  <NavigationComponent
    path="/another"
  />
  <div
    className="container"
  >
    <h1>
      Another PAge
    </h1>
  </div>
</div>
`;
